src/Command/ModifyEntity/LineItem/LineItemChangeDatesCommand.php

```diff
@@ -0,0 +1,338 @@
+ */
+
+declare(strict_types=1);
+
+namespace OAT\SimpleRoster\Command\ModifyEntity\LineItem;
+
+use Carbon\Carbon;
+use DateTime;
+use InvalidArgumentException;
+use OAT\SimpleRoster\Command\BlackfireProfilerTrait;
+use OAT\SimpleRoster\Command\Cache\LineItemCacheWarmerCommand;
+use OAT\SimpleRoster\Repository\Criteria\FindLineItemCriteria;
+use OAT\SimpleRoster\Repository\LineItemRepository;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Throwable;
+
+class LineItemChangeDatesCommand extends Command
+{
+    use BlackfireProfilerTrait;
+
+    public const NAME = 'roster:modify-entity:line-item:change-dates';
+
+    private const OPTION_LINE_ITEM_IDS = 'line-item-ids';
+    private const OPTION_LINE_ITEM_SLUGS = 'line-item-slugs';
+    private const OPTION_START_DATE = 'start-date';
+    private const OPTION_END_DATE = 'end-date';
+    private const OPTION_FORCE = 'force';
+
+    /** @var SymfonyStyle */
+    private $symfonyStyle;
+
+    /** @var bool */
+    private $isDryRun;
+
+    /** @var string[] */
+    private $lineItemSlugs;
+
+    /** @var int[] */
+    private $lineItemIds;
+
+    /** @var DateTime|null */
+    private $startDate;
+
+    /** @var DateTime|null */
+    private $endDate;
+
+    /** @var LineItemRepository */
+    private $lineItemRepository;
+
+    public function __construct(LineItemRepository $lineItemRepository)
+    {
+        parent::__construct(self::NAME);
+
+        $this->lineItemRepository = $lineItemRepository;
+    }
+
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this->addBlackfireProfilingOption();
+
+        $this->setDescription('Updates the start and end dates of line item(s).');
+        $this->setHelp(<<<'EOF'
+The <info>%command.name%</info> command changes the dates for specific line items.
+
+<comment>Not specifying a start-date or end-date option will nullify the value for the column.</comment>
+<comment>Dates are expected to be in the format: 2020-01-01T00:00:00+0000</comment>
+
+To change both start and end date of a line item using IDs:
+    <info>php %command.full_name% -i 1,2,3 --start-date <date> --end-date <date></info>
+    <info>php %command.full_name% --line-item-ids 1,2,3 --start-date <date> --end-date <date></info>
+
+To change both start and end date of a line item using slugs:
+    <info>php %command.full_name% -s slug1,slug2,slug3 --start-date <date> --end-date <date></info>
+    <info>php %command.full_name% --line-item-slugs slug1,slug2,slug3 --start-date <date> --end-date <date></info>
+EOF
+        );
+
+        $this->addOption(
+            self::OPTION_LINE_ITEM_IDS,
+            'i',
+            InputOption::VALUE_REQUIRED,
```

Comment:
It doesn't work on my case since it's and option and not an argument. Also it can't become an argument since it's not mandatory.
Tried the equivalent `InputOption::VALUE_IS_ARRAY` but it seems to work differently